### DISCUSSION TITLE
Tried to find a project from months ago, but it's missing and my domain is missing. It's still running and hosted.

### DISCUSSION THREAD

**Discussion by nervestaple** [vercel/community#6724] [Account Issue]
### Question

I have a private github repo that's been hosted on Vercel on a domain I own for at least a year. Tried to find either the Vercel project or the domain in my Vercel account, but it's missing. I still have Vercel github notifications on PRs on that repo, and the domain is still up and running, it's just that my personal Vercel account -- both projects and domains -- is now empty.

**Comment by versecafe** [comment]
@nervestaple Have you checked your hobby team?
